theme/themes/eea/collections/menu.variables

```diff
@@ -269,6 +269,9 @@
 @secondaryVerticalPointingItemMargin: 0em -@secondaryPointingBorderWidth 0em 0em;
 
 
+@secondaryPointingItemBorderBottom: @1px solid @midGray;
+@secondaryPointingItemMaxWidth: 15.626rem; 
```

Comment:
@g-stamatis be careful that your values don't add up to subpixel values
15.625rem equals 250px
15.626rem equals 250.41px which on some browsers would introduce some wrong
antialiasing.